propolis/src/hw/virtio/viona.rs

```diff
@@ -377,6 +377,7 @@ struct VionaPoller {
     dev: Weak<VirtioViona>,
 }
 impl VionaPoller {
+    #[cfg(target_os = "illumos")]
```

Comment:
Does the compiler prevent you from doing the `#cfg()` guard at the top of the impl, rather than for each function?